scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -318,22 +318,30 @@ public void initActivity(Body body) {
                                                                                       PASchedulerProperties.SCHEDULER_CLIENT_POOL_NBTHREAD.getValueAsInt(),
                                                                                       120L,
                                                                                       TimeUnit.SECONDS,
-                                                                                      new NamedThreadFactory("ClientRequestsThreadPool"));
+                                                                                      new NamedThreadFactory("ClientRequestsThreadPool",
+                                                                                                             false,
+                                                                                                             3));
```

Comment:
is it possible to make the number of threads configurable? Same for the other thread pools

Reply:
it is configurable actually (all thread pools are).  The configuration pa.scheduler.core.listener.threadnumber is stored in a static variable NB_THREADS

Reply:
3 is not the pool size, it is the thread priority